base/changes.txt

```diff
@@ -6,7 +6,13 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
-2023-08-19 Joseph Wright  <Joseph.Wright@latex-project.org>
+2023-09-05 Ulrike Fischer <Ulrike.Fischer@latex-project.org>
+	* ltproperties.dtx: 
+      First implementation of properties module
+    * ltmiscen.dtx:
+      add test to \enddocument for changes in property values.   
```

Comment:
Inconsistent indent characters: 1 tab vs multiple spaces.

It seems most changes entries are indented by 1 tab, excerpt for the one(s) added by myself, which use(s) spaces.

Reply:
Should be tabs: this is (I am told) a standard format from Emacs ...